src/pages/training.js

```diff
@@ -18,6 +18,22 @@ class TrainingPage extends Component {
     modalContent: null
   }
 
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKeyPress, false)
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKeyPress, false)
+  }
+
+  handleKeyPress = ({ keyCode }) => {
+    if (keyCode === 27) {
```

Comment:
I think you can just compare it with 'Esc' (or 'esc', not sure). But either way, I think it's more clear.

btw, 👍 for doing this, I was about to do it 😄